src/index.js

```diff
@@ -8,6 +8,7 @@ import {
 	SVG_CAMEL_CASE,
 	createComponent
 } from './lib/util.js';
+import { Signal } from '@preact/signals-core';
```

Comment:
Looks like this was accidentally left around, will need to be removed but I gotta get to bed -- I can do it tomorrow but wanted to drop a mention at least so we don't release this.